python/makeALPACAEvents.py

```diff
@@ -0,0 +1,140 @@
+        f.write("'_%s_%s'       !  [outtag] : For output file\n"%(mres,gax))
+        f.write("****************************************************************************************\n")
+        f.write("************************* Integration parameters ***************************************\n")
+        f.write("****************************************************************************************\n")
+        f.write("100000       ! [ncall] : Number of calls for preconditioning\n")
+        f.write("10           ! [itmx] : Number of iterations for preconditioning\n")
+        f.write("1d0          ! [prec] :  Relative accuracy (in %) in main run\n")
+        f.write("100000       ! [ncall1] : Number of calls in first iteration\n")
+        f.write("100000       ! [inccall] : Number of increase calls per iteration\n")
+        f.write("50           ! [itend] : Maximum number of iterations\n")
+        f.write("6            ! [iseed] : Random number seed (integer > 0)\n")
+        f.write("****************************************************************************************\n")
+        f.write("******************************* Unweighted events **************************************\n")
+        f.write("****************************************************************************************\n")
+        f.write(".true.       ! [genunw] : Generate unweighted events\n")
+        f.write("%s           ! [nev] : Number of events ( < 1000000 recommended)\n"%(nev))
+        f.write("'hepevt'     ! [erec] : Event record format ('lhe' = Les Houches, 'hepevt' = HEPEVT)\n")
+        f.write("****************************************************************************************\n")
+        f.write("*******************************  Cuts   ************************************************\n")
+        f.write(".true.       ! [gencuts] : Generate cuts - [rmin], [rmax], [dmin], [emin]\n")
+        f.write(".true.       ! [adecay] : Include ALP decay\n")
+        f.write("****************************************************************************************\n")
+        f.write("****************************************************************************************\n")
+        f.write("****************************************************************************************\n")
+
+def ntupleWrite(ctau,fn,Lmin,Lmax,startZ,endZ,SmearBeam,mres,gax):
+    Lmin        = float(Lmin)
+    Lmax        = float(Lmax)
+    startZ      = float(startZ)
+    endZ        = float(endZ)
+    SmearBeam   = float(SmearBeam)
+    wdir = os.environ['ALPACABIN']
+    with open(wdir+"/outputs/output_"+str(mres)+"_"+str(gax)+".dat","r") as fp:
+        for line in fp:
+            if "Cross section" in line:
+                line=line.split()
+                xs=line[3]+"_"+line[5]
+    fn=fn.readlines()
+    inputFile="/eos/experiment/ship/data/ALP/ntuples/"+"alp_m"+str(mres)+"_g"+str(gax)+"_xs"+str(xs)+".root"
```

Comment:
I would add $EOSSHIP here, either hardcoded or taking it from the environment. The /eos/ mountpoint is only available on EOS.

Reply:
Okey.

Reply:
Although see the other comment: it's a bad idea to *write* to EOS directly.
I would suggest creating this file in the current working directory of the script.

Reply:
Same in here, waiting for my jobs are done in HTCondor.